react-front-end/tsrc/search/components/AdvancedSearchPanel.tsx

```diff
@@ -140,6 +158,13 @@ export const AdvancedSearchPanel = ({
         >
           {languageStrings.common.action.search}
         </Button>
+        <Button
+          id={`${idPrefix}-clearBtn`}
+          onClick={() => onClear()}
```

Comment:
Redundant arrow operator. Keep in mind we have first class function support, so:

```suggestion
          onClick={onClear}
```